src/main/resources/public/partials/workerHistory.html

```diff
@@ -31,37 +31,35 @@
         </tr>
         </thead>
         <tbody>
-        <tr ng-repeat-start="history in orderedHistory = (histories.list | orderBy: 'created': true)">
-            <td>{{ history.created}}</td>
-            <td ng-if="history.request">
-                <span ng-repeat="acceptance in history.acceptances">
-                    <span ng-if="history.acceptances.length == 1 || $last">{{ acceptance.decider.name }}</span>
-                    <span ng-if="history.acceptances.length > 1 && !$last">{{ acceptance.decider.name + "," }}</span>
+        <tr ng-repeat-start="history in reversedHistories = (histories | reverse)">
+            <td>{{ history.created }}</td>
```

Comment:
delete spaces inside brackets

Reply:
why? we have spaces in the whole file... why I should delete them here?